client/state/reader/conversations/selectors/get-reader-conversation-follow-status.js

```diff
@@ -18,5 +13,5 @@ import 'state/reader/init';
  * @returns {string|null} Conversation follow status (F for following, M for muting, or null)
  */
 export default function getReaderConversationFollowStatus( state, { siteId, postId } ) {
-	return get( state, [ 'reader', 'conversations', 'items', key( siteId, postId ) ], null );
+	return state?.reader?.conversations?.items?.[ key( siteId, postId ) ] ?? null;
```

Comment:
Assuming that `reader` reducer has been loaded, `conversations` and its `items` will always exist.  Also, what is the chance that we'll call the selector with a `null` or `undefined` `state`?